src/components/StackedBarChart/mapStackedBarchartData.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  DailyWaterAmountsType,
+  RadolanDays,
+  WateredDayType,
+} from '../../common/types';
+
+const _radolanDaysToHours = (radolanDays: RadolanDays): number[] => {
```

Comment:
Ok. I usually never prefix anything with an underscore. I did this here because you did so as we did the live share session and it was also inside a module. So in which cases do you use the underscore as a prefix?

Reply:
I only prefix with _ if the function of not used to silence eslint.

Reply:
Oh ok I thought it was to indicate as "private" :) I will remove the underscores then.